index.html

```diff
@@ -1,413 +1,259 @@
-                          src="./images/HTML.png"
-                          alt="html-icon"
-                        />HTML</span
-                      >
-                    </li>
-                    <li>
-                      <span
-                        ><img src="./images/CSS.png" alt="css-icon" />CSS</span
-                      >
-                    </li>
-                  </ul>
-                </div>
-              </div>
-            </li>
-            <li>
-              <div class="skill-projects-item-one">
-                <p>
-                  Frameworks<span
-                    ><img src="./images/arrow.png" alt="frames-icon"
-                  /></span>
-                </p>
-              </div>
-              <hr/>
-            </li>
-            <li>
-              <div class="skill-projects-item-two">
-                <p>
-                  Skills<span
-                    ><img src="./images/arrow.png" alt="skills-icon"
-                  /></span>
-                </p>
-              </div>
-              <hr/>
-            </li>
-          </ul>
+        <div>
+          <button type="button" class="button-resume"><span>Get my Resume</span></button>
         </div>
       </section>
-    </section>
-    <section class="form">
-      <form action="https://formspree.io/f/xeqndgga" method="post">
-        <h1>Contact Me</h1>
-        <p>
-          If you have an application you<br />are interested in developing, a<br />feature
-          that you need built or a<br />project that needs coding. I’d<br />
-          love to help with it
-        </p>
-        <ul class="form-sections">
+    <section class="skill-section-items">
+    <ul>
+      <li>
+        <div class="languages-dropdown-bar">
+        <p>Langauges<span><img src="./images/dropdown..png" alt="languages"></span></p>
+        <div class="skill-items">
+        <ul>
           <li>
-            <label for="name"></label>
-            <input
-              type="text"
-              id="name"
-              name="user_name"
-              maxlength="30"
-              placeholder="Name "
-              required
-            />
-          </li>
+            <span><img src="./images/JS.png" alt="js-icon">Javascript</span></li>
           <li>
-            <label for="mail"></label>
-            <input
-              type="email"
-              id="mail"
-              name="user_mail"
-              placeholder="Your email "
-              required
-            />
-          </li>
+          <span><img src="./images/HTML.png" alt="html-icon">HTML</span></li>
           <li>
-            <label for="msg"></label>
-            <textarea
-              class="textarea"
-              id="msg"
-              name="user_message"
-              maxlength="500"
-              placeholder="Write your message here"
-              required
-            ></textarea>
+            <span><img src="./images/CSS.png" alt="css-icon">CSS</span></li>
+          </ul>
```

Comment:
- [ ] As the previous reviewer mentioned kindly align this section properly so it looks similar to the Figma design.

|Your design|Figma design|
|-|-|
|![Screenshot_20220610_181145](https://user-images.githubusercontent.com/25227022/173117418-6cea2c1d-d938-4ab5-81cd-9e076067f22d.png)|![173090959-4726e422-35d7-4811-9861-9191a9879d3d](https://user-images.githubusercontent.com/25227022/173117472-35d18acd-4033-4003-915d-91d0525a5861.png)|